### PR TITLE
Add -NoProfile flag to windows_browser_apps_to_cmd

### PR DESCRIPTION
**Pull request by np-8**
- Related: https://github.com/SergeyPirogov/webdriver_manager/pull/321
- Fixes: https://github.com/SergeyPirogov/webdriver_manager/issues/320

### CODE REVIEW

File: webdriver_manager/utils.py [hunk 1]
```diff
@@ -148,7 +148,7 @@ def windows_browser_apps_to_cmd(*apps: str) -> str:
         ''.join(f" if (-not $? -or $? -match $error) {{ {i}{ignore_errors_cmd_part} }}" for i in apps[1:])
     )
 
-    return f"{powershell}{script}"
+    return f"{powershell} -NoProfile {script}"
```
Comment (aleksandr-kotlyar) [inline webdriver_manager/utils.py:151]: @np-8 is it ok to add extra-space before "-NoProfile":
`powershell  -NoProfile {script}`
when it's not powershell console
or
` -NoProfile {script}`
when it's running in powershell already?

Comment (aleksandr-kotlyar) [reply webdriver_manager/utils.py:151]: May be add `"-NoProfile"` into the `${script}` variable?

Comment (aleksandr-kotlyar) [reply webdriver_manager/utils.py:151]: @np-8 May be add `"-NoProfile"` into the `${script}` variable?

**Review by aleksandr-kotlyar — CHANGES_REQUESTED** [review]

File: webdriver_manager/utils.py (hunk 1 above)
Comment (np-8) [reply webdriver_manager/utils.py:151]: @aleksandr-kotlyar I tested (in `cmd` and `Powershell`)  and it really does not matter if there are extra space in any part of the command. I.e. this works fine:

```
C:\Users\Niko>  powershell      -NoProfile   ls  
```

The console used to launch the `powershell` should not affect the number of parameters it takes, so yes to that. I checked the versions and `-NoProfile` is [listed in the oldest (5.1) Powershell help](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_powershell_exe?view=powershell-5.1). It seems that only the `-Mta` parameter has been added to the available parameters (in Powershell 3.0).

The syntax for the call is 

```
powershell <parameters> -Command <script>
```

or 

```
powershell <parameters>  <script>
```

So if the `-NoProfile` flag/parameter is wanted to be separately, I would suggest

```
return f"{powershell} {parameters} {script}"
```

Comment (aleksandr-kotlyar) [reply webdriver_manager/utils.py:151]: @np-8 thank you for answer! Okay, number of spaces doesn't matter - that's good.

Comment (aleksandr-kotlyar) [reply webdriver_manager/utils.py:151]: No need to separate params from the command - that's why I think it's better to keep them together in `script` variable for now.

**Pull request merged**